### PR TITLE
Fix [=nested browsing context=]

### PR DESCRIPTION
**Pull request by eladalon1983**
Editorial.

***
<a href="https://pr-preview.s3.amazonaws.com/eladalon1983/mediacapture-viewport/pull/29.html" title="Last updated on Sep 26, 2024, 10:16 PM UTC (42edcbe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-viewport/29/eed9b8d...eladalon1983:42edcbe.html" title="Last updated on Sep 26, 2024, 10:16 PM UTC (42edcbe)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -270,7 +270,7 @@ <h2>
                     documents that consist of the
                     [=relevant global object=]'s [=associated `Document`=]'s
                     [=top-level browsing context=]'s <a data-cite="HTML#nav-document">active document</a>, and
-                    all <a data-cite="HTML#nav-document">active documents</a> in [=nested browsing context=]s of
+                    all <a data-cite="HTML#nav-document">active documents</a> in nested [=browsing context=]s of
```
Comment (dontcallmedom) [inline index.html:273]:
```suggestion
                    all [=navigable/active documents=] in nested [=browsing context=]s of
```

Comment (eladalon1983) [reply index.html:273]: I'm going to make that additional change in a separate PR, for easier tracking.

**Comment by eladalon1983** [comment]
FYI @jan-ivar.

File: index.html (hunk 1 above)
Comment (eladalon1983) [reply index.html:273]: Done: https://github.com/w3c/mediacapture-viewport/pull/30

**Pull request merged**